include/ignition/rendering/base/BaseAxisVisual.hh

```diff
@@ -37,6 +37,18 @@ namespace ignition
       public: virtual ~BaseAxisVisual();
 
       public: virtual void Init();
+
+      public: virtual void SetLocalScale(double _x, double _y, double _z) override;
+
+      public: virtual void SetLocalScale(double _scale) override;
+
+      public: virtual void SetLocalScale(const math::Vector3d &_scale) override;
```

Comment:
I think you should be able to just override this version with `math::Vector3d` arg. The other two just calls this function so can be removed.